ios/FluentUI/Controls/IndeterminateProgressBarView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@available(*, deprecated, renamed: "IndeterminateProgressBarView")
+public typealias MSIndeterminateProgressBarView = IndeterminateProgressBarView
+
+/**
+Indeterminate progress bar view
+ */
+@objc(MSFIndeterminateProgressBarView)
+open class IndeterminateProgressBarView: UIView {
+	
+	/// progressbar's track
+	var trackLayer = CALayer()
+
+	/// gradient layer that slides across the track
+	var gradientLayer = CAGradientLayer()
+	
+	@objc override public init(frame: CGRect) {
+		super.init(frame: frame)
+	}
+	
+	required public init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	open override func layoutSubviews() {
```

Comment:
>layoutSubviews [](start = 20, length = 14)

are you missing overriding  func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) to listen to darkmode/lightmode changes to update the cgcolor of the gradient layer?